webapp/src/app/[locale]/page.tsx

```diff
@@ -1,6 +1,6 @@
 'use client'
 
-import { useSuspenseQuery } from '@apollo/experimental-nextjs-app-support/ssr'
+import { useQuery } from '@apollo/experimental-nextjs-app-support/ssr'
```

Comment:
`useSuspenseQuery` after compiling, it returned an error blocking the webapp to not load correctly.